cmd/main.go

```diff
@@ -37,6 +38,7 @@ func main() {
 		),
 		cli.WithDefaultPlugins(cfgv2.Version, &pluginv2.Plugin{}),
 		cli.WithDefaultPlugins(cfgv3alpha.Version, &pluginv3.Plugin{}),
+		cli.WithDefaultPlugins(cfgv3.Version, &pluginv3.Plugin{}),
```

Comment:
Will not we replace the cfgv3alpha for cfgv3? I think also it can be the next follow-up after this one. WDYT?

Reply:
It can be done either ina follow up or in this PR, but for now I left both so that we can start testing things.

Reply:
IHMO, if you add both than it means more work do, +1 e2e test + generate data and etc .. as it will make harder to the reviewers. So, if you close the scope here for it does one 1 proposed idea whcih is store in the PROJECT file the attrs fields I think we can move smothier.